lowcar/devices/KoalaBear/KoalaBear.h

```diff
@@ -35,12 +35,12 @@ class KoalaBear : public Device {
     virtual void device_actions();
 
   private:
-    float target_speed_a, target_speed_b;  // target speeds of motors
-    float deadband_a, deadband_b;          // deadbands of motors
-    uint8_t pid_enabled_a, pid_enabled_b;  // whether or not motors are enabled
-    volatile uint32_t enc_a, enc_b;        // encoder values of motors
-    PID *pid_a, *pid_b;                    // PID controllers for motors
-    LEDKoala* led;                         // for controlling the KoalaBear LED
+    float target_speed_a, target_speed_b;           // target speeds of motors
+    float deadband_a, deadband_b;                   // deadbands of motors
+    uint8_t invert_a, invert_b;                     // whether or not motors are going in default direction
+    uint8_t manual_enabled_a, manual_enabled_b;     // whether or not PID on motors is enabled
```

Comment:
These comments might need to be updated
I like the wording used in `KoalaBear.cpp`:
```
// TRUE if the respective motor should rotate in opposite direction of default; FALSE for default direction
// TRUE if using manual drive mode; FALSE if using PID control (default)
```

Reply:
Yes, fixed--although the `manual_enabled_a` stuff might need to be updated (see later comment)